deploy/manifests/base/assigner/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: assigner
+spec:
+  # Do not run more than one replica; this overlay is designed to run a single instance only.
+  replicas: 1
+  selector:
+    matchLabels:
+      app: assigner
+  template:
+    metadata:
+      labels:
+        app: assigner
+    spec:
+      securityContext:
+        runAsUser: 10001
+        fsGroup: 532
+      containers:
+        - name: assigner
+          image: storetheindex
+          env:
+            - name: GOLOG_LOG_LEVEL
+              value: INFO
+            - name: GOLOG_LOG_FMT
+              value: json
+            - name: ASSIGNER_PRIV_KEY_PATH
+              value: /identity/identity.key
+            - name: ASSIGNER_PATH
+              value: /config
+          ports:
+            - containerPort: 3001
+              name: http
+            - containerPort: 8081
+              name: metrics
+          volumeMounts:
+            - name: identity
+              mountPath: /identity
+            - name: config
+              mountPath: /config
```

Comment:
`config` volume no longer exists; this needs to be removed too.

Reply:
Done